# HG changeset patch
# User Rik <rik@octave.org>
# Date 1385447533 28800
#      Mon Nov 25 22:32:13 2013 -0800
# Branch stable
# Node ID 683f9c39da0680117eecd38aa807754cae985ecf
# Parent  501cb6a90f0ff306db7023e25c0495c4aa64001f
recursive_nest2.m: Replace tab with space.

diff --git a/test/nest/recursive_nest2.m b/test/nest/recursive_nest2.m
--- a/test/nest/recursive_nest2.m
+++ b/test/nest/recursive_nest2.m
@@ -3,19 +3,19 @@ function x = recursive_nest2 ()
   x = B (20);
   function v = B (n)
     Y = 0;
     BB (n);
     C;
     v = Y;
     function BB (m)
       if m > 0
-	Y = Y + 1;
-	BB(m - 1);
-	C;
+        Y = Y + 1;
+        BB(m - 1);
+        C;
       end
     endfunction
   endfunction
 
   function C
     Y = 0;
   endfunction
 endfunction
